tests/robot-tests/tests/public_api/public_api_minor_manual_changes.robot

```diff
@@ -0,0 +1,386 @@
+    user waits until h2 is visible    API data set quick start
+    user waits until h2 is visible    API data set version history
+    user waits until h2 is visible    API data set changelog
+
+User verifies the public data guidance in the 'API data set changelog' section
+    user waits until element contains    testid:public-guidance-notes    Content for the public guidance notes
+
+User verifies minor changes in the 'API data set changelog' section
+    user waits until h3 is visible    Minor changes for version 1.1
+    ${minor_changes_section}=    get child element    id:apiChangelog    testid:minor-changes
+
+    ${school_types_filter}=    user checks changelog section contains updated filter    ${minor_changes_section}
+    ...    School type
+    ${updated_school_types_total}=    user checks changed facet contains option    ${school_types_filter}    Total
+    user checks changed option contains description    ${updated_school_types_total}
+    ...    label changed to: State-funded primary and secondary
+    user checks changed option contains description    ${updated_school_types_total}    no longer an aggregate
+
+    ${updated_regional_options}=    user checks changelog section contains updated location level
+    ...    ${minor_changes_section}    Regional
+    ${updated_yorkshire_and_humber}=    user checks changed facet contains option    ${updated_regional_options}
+    ...    Yorkshire and The Humber
+    user checks changed option contains description    ${updated_yorkshire_and_humber}    label changed to: Yorkshire
+
+User verifies no major changes are present
+    user checks page does not contain element    testid:major-changes
+
+
+*** Keywords ***
+user checks changelog section contains updated filter
+    [Arguments]
+    ...    ${changes_section}
+    ...    ${filter_label}
+    ${filter}=    user checks element contains child element    ${changes_section}
+    ...    xpath://div[starts-with(@data-testid, "updated-filterOptions")][h4[contains(., "Updated ${filter_label} filter options")]]
+    RETURN    ${filter}
+
+user checks changelog section contains updated location level
+    [Arguments]
+    ...    ${changes_section}
+    ...    ${level_label}
+    ${location_level}=    user checks element contains child element    ${changes_section}
+    ...    xpath://div[starts-with(@data-testid, "updated-locationOptions")][h4[contains(., "Updated ${level_label} location options")]]
+    RETURN    ${location_level}
+
+user checks changed facet contains option
+    [Arguments]
+    ...    ${changed_facet}
+    ...    ${option_label}
+    ${option}=    user checks element contains child element    ${changed_facet}
+    ...    xpath://li[@data-testid="updated-item" and contains(., "${option_label}")]
+    RETURN    ${option}
+
+user checks changed option contains description
+    [Arguments]
+    ...    ${changed_option}
+    ...    ${description}
+    user checks element contains    ${changed_option}    ${description}
```

Comment:
Do we need this keyword? It's pretty redundant given it's just doing a `user checks element contains`

Reply:
Nah, happy to remove - I do sometimes like to have a keyword that's more descriptive of what you're actually looking for, but I think this is OK to remove.